FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/include/FreeRTOS_Sockets.h

```diff
@@ -203,6 +203,7 @@ extern const char *FreeRTOS_inet_ntoa( uint32_t ulIPAddress, char *pcBuffer );
 /* The socket type itself. */
 struct xSOCKET;
 typedef struct xSOCKET *Socket_t;
+
```

Comment:
What changed here and at line 368?  Can we undo these?

Reply:
Done, thanks